connector/third_party/apache-spark/src/main/java/org/apache/spark/bigquery/BigNumericUDT.scala

```diff
@@ -0,0 +1,38 @@
+package org.apache.spark.bigquery
```

Comment:
Can you please port to Java, as we will need BigNumeric support in the DSv2 java only implementations as well